gravitee-am-service/src/main/java/io/gravitee/am/service/DomainService.java

```diff
@@ -36,12 +36,14 @@
  */
 public interface DomainService {
 
+    Flowable<Domain> findAllByEnvironment(String organizationId, String environment, int page, int size);
+
+    Flowable<Domain> search(String environmentId, String query, int page, int size);
```

Comment:
I think both page and size are not used in the service layer. We can remove them.

Reply:
resolved